clang/lib/Driver/ToolChains/SYCL.cpp

```diff
@@ -1008,11 +1008,20 @@ SYCLToolChain::SYCLToolChain(const Driver &D, const llvm::Triple &Triple,
   getProgramPaths().push_back(getDriver().Dir);
 
   // Diagnose unsupported options only once.
-  // All sanitizer options are not currently supported.
-  for (auto A :
-       Args.filtered(options::OPT_fsanitize_EQ, options::OPT_fcf_protection_EQ))
+  // All sanitizer options are not currently supported, except AddressSanitizer
+  for (auto *A : Args.filtered(options::OPT_fsanitize_EQ,
+                               options::OPT_fcf_protection_EQ)) {
+    if (A->getOption().getID() == options::OPT_fsanitize_EQ) {
+      if (A->getValues().size() == 1) {
+        std::string SanitizeVal = A->getValue();
+        if (SanitizeVal == "address") {
```

Comment:
According to https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements.
Brace is not needed here

Reply:
Thanks, Will do